seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4198,11 +4198,11 @@
     },
     {
       "id": "56533eb9ac21ba0edf2244c7",
-      "title": "Accessing Object Properties with the Dot Operator",
+      "title": "Accessing Object Properties with the Dot Notation",
       "description": [
-        "There are two ways to access the properties of an object: the dot operator (<code>.</code>) and bracket notation (<code>[]</code>), similar to an array.",
-        "The dot operator is what you use when you know the name of the property you're trying to access ahead of time.",
-        "Here is a sample of using the dot operator (<code>.</code>) to read an object's property:",
+        "There are two ways to access the properties of an object: the dot notation (<code>.</code>) and bracket notation (<code>[]</code>), similar to an array.",
+        "The dot notation is what you use when you know the name of the property you're trying to access ahead of time.",
```

Comment:
I think we should remove "the" from these even though MDN does refer to it as "the dot notation" because we don't refer to bracket notation as "the bracket notation". In other words, it should simply be "dot notation" instead of "the dot notation". Thoughts cc @freeCodeCamp/moderators

Reply:
@dhcodes I like removal :+1: It does sound a bit unnatural so I'm all for remote the "the" in front of "dot notation".

Reply:
thanks for the second opinion @erictleung!

@Bouzmine if you still have the branch, navigate to it, remove `the` in from of dot notation in these paragraphs, save the file, do a `git add <filename>`, and then do `git commit --amend --no-edit`. Finally do a `git push origin fix/dot-notation -f` to force-push the changes. 

Thanks.